### PR TITLE
bugfix/#2911-edit-profile-layout-fix

### PR DESCRIPTION
**Pull request by dyadyushko**
Lining up content in profile edit page to global container

1440px

1024px

768px

425px

320px

### CODE REVIEW

File: src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss [hunk 1]
```diff
@@ -35,8 +39,11 @@ input:checked ~ .checkmark::after {
   display: block;
 }
 
-.container {
-  margin-left: 135px;
+.edit_prof-container {
+  width: 100%;
+}
```
Comment (Edw1x) [inline src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss:44]: If we will remove these lines of code, will something change?

Comment (dyadyushko) [reply src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss:44]: this outer wrapper is serving for taking all the available space from bootstrap container, so we can allocate our content at the very left side of it

**Review by Anastasiia-Hevchuk — APPROVED** [review]

**Review by Edw1x — APPROVED** [review]

File: src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss [hunk 2]
```diff
@@ -1,3 +1,7 @@
+.container {
+  padding: 0;
+}
```
Comment (JimHolman1199) [inline src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss:3]: let's not rewrite the bootstrap class, but use p-0 class instead 
https://getbootstrap.com/docs/4.0/utilities/spacing/

**Review by AlekOK — APPROVED** [review]

**Review by YuriShevchuk — APPROVED** [review]

File: src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss (hunk 2 above)
Comment (dyadyushko) [reply src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss]: done

**Review by YulianDev — APPROVED** [review]

**Review by tkhristina — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

**Comment by JimHolman1199** [comment]
resolve conflicts so we can mege this

**Pull request closed without merging**